### PR TITLE
fix: transfer page buttons alignment

### LINKED ISSUE #9530: Transfer page tab buttons
### What happened?

CleanShot 2024-02-23 at 08 58 21@2x

### PR DESCRIPTION
**Pull request by iMac7**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [ ] Closes #9530
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw&amount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot_20240223_085434

### CODE REVIEW

**Comment by hassnian** [comment]
@iMac7 pls also center them

**Comment by iMac7** [comment]
done

**Review by hassnian — COMMENTED** [review]
the root cause of the issue , some other buttons might also be broken 

CleanShot 2024-02-23 at 17 16 22@2x

**Review by hassnian — APPROVED** [review]

**Comment by Jarsen136** [comment]
> the root cause of the issue , some other buttons might also be broken
> 
> CleanShot 2024-02-23 at 17 16 22@2x


Maybe we should fix the root issue instead of adding a fix every place.
image

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
.

**Comment by iMac7** [comment]
> the root cause of the issue , some other buttons might also be broken
> 
> CleanShot 2024-02-23 at 17 16 22@2x

nice catch, any reason it might have been commented out? I hope undoing this doesn't break anything :eyes:

**Comment by hassnian** [comment]
@iMac7 check any pill variant used inside that pr and see if you can add a local fix there

**Comment by iMac7** [comment]
> @iMac7 check any pill variant used inside that pr and see if you can add a local fix there
Just uncommenting [this](https://github.com/kodadot/nft-gallery/pull/9394/files#diff-6cac81a09ae16a212a724a6359cce50c6031ffb32b5746b2fc6c34c2164e93e0) seems to fix things, are you suggesting I leave the comment as it is and apply local fixes on the pill buttons?

File: libs/ui/src/components/NeoButton/NeoButton.scss [hunk 1]
```diff
@@ -178,11 +178,9 @@
         @apply bg-background-color-inverse text-text-color-inverse border border-background-color-inverse;
       }
 
-      // commented out in https://github.com/kodadot/nft-gallery/pull/9394
```
Comment (Jarsen136) [inline libs/ui/src/components/NeoButton/NeoButton.scss:181]: You should check what causing the issue #9394 here and fix it.

**Comment by daiagi** [comment]
Oh yes.
I commented that line out
It was causing wierd behviour on code Checker pill variant buttons

So whatever fix is. Please check that buttons over there stay centered
Pay attention to button that have icon as well

https://canary.kodadot.xyz/code-checker

**Comment by iMac7** [comment]
> Oh yes. I commented that line out It was causing wierd behviour on code Checker pill variant buttons
> 
> So whatever fix is. Please check that buttons over there stay centered Pay attention to button that have icon as well
> 
> https://canary.kodadot.xyz/code-checker

it was happening on the pill variants, any style that overrides a similar style in `NeoButton.scss` is not applied

**Pull request closed without merging**